app/helpers/statistics_helper.rb

```diff
@@ -1,4 +1,37 @@
 module StatisticsHelper
   # Module for generating and collecting statistics.
   # Please let your helper function start with "statistics_"
+
+  def statistics_item_popularity(item)
```

Comment:
Do you want to link the wiki page here too? :)

Reply:
Done :)